content/zh/docs/ops/best-practices/image-signing-validation/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: 图像签名和验证
+description: 描述如何使用镜像签名来验证 Istio 镜像的出处。
+weight: 35
+aliases: []
+keywords: [install,signing]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+本页介绍如何使用 [Cosign](https://github.com/sigstore/cosign) 验证 Istio 镜像制品的来源。
+
+Cosign 是作为 [sigstore](https://www.sigstore.dev) 项目的一部分开发的工具。
+它简化了已签名的开放容器倡议（OCI）制品（例如容器镜像）的签名和验证。
+
+从 Istio 1.12 开始，我们签署所有正式发布的容器镜像作为我们发布过程的一部分。
+然后，最终用户可以使用下面描述的过程来验证这些镜像。
+
+此过程适用于手动执行或与构建或部署管道集成以自动验证镜像制品。
+
+
+## 先决条件{#prerequisites}
+
+在开始之前，请执行以下操作：
+
+1. 为您的架构下载最新的 [Cosign](https://github.com/sigstore/cosign/releases/latest) 构建及其签名。
+1. 验证 `cosign` 二进制签名：
+
+   {{< text bash >}}
+$ openssl dgst -sha256 \
+    -verify <(curl -ssL https://raw.githubusercontent.com/sigstore/cosign/main/release/release-cosign.pub) \
+    -signature <(cat /path/to/cosign.sig | base64 -d) \
+    /path/to/cosign-binary
+    {{< /text >}}
+
+1. 使二进制文件可执行（'chmod + x'），并移动到 'PATH' 上的一个位置。
```

Comment:
Need to run this locally to see the highlighting. Not sure if it's doing something with the comma or is it a "`" va "'" on the line.

Reply:
I have try to `make lint`,However it appears as follows


<img width="1127" alt="image" src="https://user-images.githubusercontent.com/29862632/188930655-c872b057-fe6b-403e-9570-45c66de88e49.png">

Reply:
You need to wait longer for the actual lint messages. Sample output for another PR.
![Screen Shot 2022-09-07 at 11 41 47 AM](https://user-images.githubusercontent.com/10537847/188933577-a374be5e-24a7-4cd5-90db-2fe309e5c0ea.png)

Reply:
<img width="1020" alt="image" src="https://user-images.githubusercontent.com/29862632/188936189-0a217087-a308-482c-9583-ab733ffd06ab.png">